TechnicoBackend/Services/PropertyService.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TechnicoBackend.Models;
+using TechnicoBackend.DTOs;
+using TechnicoBackend.Repositories;
+
+namespace TechnicoBackend.Services
+{
+    public class PropertyService
+    {
+        private readonly PropertyRepository _propertyRepository;
```

Comment:
Consider initializing an IPropertyRepository here instead of PropertyRepository, which would be mapped accordingly using proper Dependency Injection in Program.cs.